doc/api_assets/style.css

```diff
@@ -780,6 +780,7 @@ kbd {
 }
 
 .js-flavor-selector {
+  -webkit-appearance: unset;
```

Comment:
why not `none`, like the unprefixed one?

Reply:
no preference here, can match it

Reply:
```suggestion
  -webkit-appearance: none;
```